packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -238,25 +247,29 @@ describe('basic grid', () => {
 
 describe('with pinned columns', () => {
   it('sets pinnedColumns as pinned', () => {
+    const pinnedColumns = dataFile.cols.slice(0, 2);
```

Comment:
Since we're pulling out the `pinnedColumns` into its own variable, can we do the same with `overflowColumns` since they're slicing the same base array? Also, is it possible to move them up a block into the `describe` so that the variables can be shared by each test and don't need to be redeclared in each `it`?

Reply:
These existing tests broke because we added 1 additional row to the test data. We only made this simple tweak to make the tests less fragile. Should a similar change be necessary in the future, these tests will no longer break.